app/src/main/res/values-nb/strings.xml

```diff
@@ -1,24 +1,38 @@
+    <string name="raise_to_answer_is_listening_to_sensor_data">Raise To Answer, lytter på sensor data</string>
     <string name="privacy_policy">Personvern</string>
-    <string name="could_not_bind_sensor">Klarte ikke å regisrere sensorene på enheten, din enhet mangler nærhet sensor eller akselerometer. Løft opp for å svare kan ikke fortsette</string>
+    <string name="could_not_bind_sensor">Klarte ikke detektere sensorer på enheten din. Raise To Answer kan ikke fortsette</string>
     <string name="feature_raise_to_answer">Svar ved å holde telefonen opp til ditt øre</string>
     <string name="select_features">Vennligst velg de egenskapene du vil skru på</string>
-    <string name="feature_flip_over_to_decline">Ikke svar ved snu mobilen med skjermen ned først (krever Android 9 eller nyere og et magnetometer )</string>
+    <string name="feature_flip_over_to_decline">Ikke svar ved snu mobilen med skjermen ned (krever Android 9 eller nyere og et magnetometer )</string>
     <string name="test_result">Test resultater</string>
     <string name="detected_raise_to_answer">Godta respons gjenkjent.</string>
     <string name="detected_flip_over">Leggpå respons gjenkjent</string>
     <string name="test_started">Test startet</string>
-    <string name="enable_at_least_one_feature">Vennligst skru på hvertfall en egenskap</string>
+    <string name="enable_at_least_one_feature">Vennligst skru på minst en egenskap</string>
     <string name="select_behaviour">Vennligst velg hvordan du vil at applikasjonen skal oppføre seg.</string>
-    <string name="behaviour_beep">Pip på innkommende samtaler</string>
+    <string name="behaviour_beep"> Lag lyd på innkommende samtaler</string>
     <string name="raise_enabled_key" translatable="false">raise_enabled_key</string>
     <string name="flip_over_enabled_key" translatable="false">flip_over_enabled_key</string>
     <string name="beep_behaviour_enabled_key" translatable="false">beep_behaviour_enabled_key</string>
     <string name="could_not_bind_magnetometer">Din telefon ser ut til å mangle et magnetometer. Løft opp for å svare vil ikke kunne detektere nøyaktigt orientering av telefonen.</string>
-</resources>
\ No newline at end of file
+    <string name="feature_answer_all_angles">Ta telefonen i alle vinkler </string>
+
+
+    <string name="decline_holder">Legg på  </string>
+    <string name="decline_holder">Declining calls</string>
+
+    <string name="answer_holder">Answering calls</string>
+
+    <string name="answer_all_angles_enabled_key" translatable="false">answer_all_angles_enabled_key</string>
+
+    <string name="missing_support_magnetometer">Your device lacks a magnetometer, declining or limiting answering to an up-right angle is not supported.</string>
+
+    <string name="missing_support_android_9">Your device is running a version of Android before 9, declining calls is not supported.</string>
+</resources>
+</resources>
```

Comment:
Can you remove this duplicate </resources> tag?

Reply:
ah yes :)